pages/common/tgpt.md

```diff
@@ -0,0 +1,37 @@
+# tgpt
+
+> A terminal-based AI chatbot without the need for API keys.
```

Comment:
```suggestion
> Talk to many AI chatbots without the need for API keys.
```
We generally avoid saying the program is terminal/CLI-based, it's rather better to empathisize when they are GUI-based. See [style guide#descriptions](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions):

> ## Descriptions
[...] 
> - Avoid mentioning that the program is used on the command-line (e.g. use `Ripgrep is a recursive line-oriented search tool` instead of `Ripgrep is a recursive line-oriented CLI search tool`).

See [style guide#imperative-mood](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood):

> ### Imperative Mood
> 
> - **All descriptions must be concise and phrased in the imperative mood.**
> - This also applies to all translations by default unless otherwise specified in the language-specific section below.
> - For example, when writing documentation for `cd`, a tool to check out and work on a specific directory in the Terminal or Command Prompt, **do not** write a lengthy description such as:
> 
> ```md
> > `cd` is a system tool, available in Windows, macOS, and Linux, to check out a specific directory to get things done in > the Command Prompt, Terminal, and PowerShell.
> ```
> 
> It should instead be simplified to make it easier for everyone to read:
> 
> ```md
> > Change the current working directory.
> ```

Feel free to reword it.